inc/utility/namespace.php

```diff
@@ -1500,22 +1500,3 @@ function do_shortcode_by_tags( $content, array $tags ) {
 	$shortcode_tags = $_tags;
 	return $shortcoded;
 }
-
-function initialize_sentry() {
```

Comment:
Is this possible to reuse also the existing sentry environment variables so it's backward compatible?

Reply:
yes, is possible. I can add that compatibility if it is convenient for you. Probably I would need document the reasons for future references. Could you help me with that?